chef/cookbooks/ceph/recipes/radosgw.rb

```diff
@@ -68,6 +49,16 @@
   subscribes :restart, "template[/etc/ceph/ceph.conf]"
 end
 
+# In the systemd case, need extra targets enabled
+service "ceph-radosgw.target" do
+  action :enable
+  only_if { File.exist?("/usr/lib/systemd/system/ceph-radosgw.target") }
```

Comment:
Hrm, okay. I guess another way is to just put that inside a platform family check for "suse". But no big deal.